apps/src/lib/tools/jsinterpreter/JSInterpreter.js

```diff
@@ -1063,6 +1063,7 @@ export default class JSInterpreter {
     // NOTE: we are being a little tricky here (we are re-running
     // part of the Interpreter constructor with a different interpreter's
     // scope)
+    evalInterpreter.global = this.interpreter.global;
```

Comment:
It is important that the `evalInterpreter` instance has the same `.global` scope set (since we are overriding what would normally be its separate scope). This fixes using the console to look at global custom marshal properties like `mouseX`